Firmware/mmu2.cpp

```diff
@@ -482,10 +482,6 @@ void MMU2::UnloadInner() {
         IncrementMMUFails();
     }
     MakeSound(Confirm);
-
-    // no active tool
-    SetCurrentTool(MMU2_NO_TOOL);
-    tool_change_extruder = MMU2_NO_TOOL;
 }
```

Comment:
What exactly does this change fix? Steps to reproduce would help describe it. I am worried this change introduces bugs. If this is indeed a bug, then it's likely its on the MK4 side too.

Reply:
Added the steps to test it.